// partners/fesm5/partners.js
import { Injectable, Component, NgModule, defineInjectable } from '@angular/core';

/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,uselessCode} checked by tsc
 */
var PartnersService = /** @class */ (function () {
    function PartnersService() {
    }
    PartnersService.decorators = [
        { type: Injectable, args: [{
                    providedIn: 'root'
                },] },
    ];
    /** @nocollapse */
    PartnersService.ctorParameters = function () { return []; };
    /** @nocollapse */ PartnersService.ngInjectableDef = defineInjectable({ factory: function PartnersService_Factory() { return new PartnersService(); }, token: PartnersService, providedIn: "root" });
    return PartnersService;
}());

/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,uselessCode} checked by tsc
 */
var PartnersComponent = /** @class */ (function () {
    function PartnersComponent() {
    }
    /**
     * @return {?}
     */
    PartnersComponent.prototype.ngOnInit = /**
     * @return {?}
     */
    function () {
    };
    PartnersComponent.decorators = [
        { type: Component, args: [{
                    selector: 'lib-partners',
                    template: "\n    <p>\n      partners works!\n    </p>\n  ",
                    styles: []
                },] },
    ];
    /** @nocollapse */
    PartnersComponent.ctorParameters = function () { return []; };
    return PartnersComponent;
}());

/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,uselessCode} checked by tsc
 */
var NavigationComponent = /** @class */ (function () {
    function NavigationComponent() {
    }
    /**
     * @return {?}
     */
    NavigationComponent.prototype.ngOnInit = /**
     * @return {?}
     */
    function () {
    };
    NavigationComponent.decorators = [
        { type: Component, args: [{
                    selector: 'lib-navigation',
                    template: "<p>\n  navigation works!\n</p>\n",
                    styles: [""]
                },] },
    ];
    /** @nocollapse */
    NavigationComponent.ctorParameters = function () { return []; };
    return NavigationComponent;
}());

/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,uselessCode} checked by tsc
 */
var PartnersModule = /** @class */ (function () {
    function PartnersModule() {
    }
    PartnersModule.decorators = [
        { type: NgModule, args: [{
                    imports: [],
                    declarations: [
                        PartnersComponent,
                        NavigationComponent
                    ],
                    exports: [
                        PartnersComponent,
                        NavigationComponent
                    ]
                },] },
    ];
    return PartnersModule;
}());

/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,uselessCode} checked by tsc
 */

/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,uselessCode} checked by tsc
 */

export { PartnersService, PartnersComponent, NavigationComponent, PartnersModule };
