// partners/bundles/partners.umd.js
(function (global, factory) {
    typeof exports === 'object' && typeof module !== 'undefined' ? factory(exports, require('@angular/core')) :
    typeof define === 'function' && define.amd ? define('partners', ['exports', '@angular/core'], factory) :
    (factory((global.partners = {}),global.ng.core));
}(this, (function (exports,i0) { 'use strict';

    /**
     * @fileoverview added by tsickle
     * @suppress {checkTypes,extraRequire,uselessCode} checked by tsc
     */
    var PartnersService = (function () {
        function PartnersService() {
        }
        PartnersService.decorators = [
            { type: i0.Injectable, args: [{
                        providedIn: 'root'
                    },] },
        ];
        /** @nocollapse */
        PartnersService.ctorParameters = function () { return []; };
        /** @nocollapse */ PartnersService.ngInjectableDef = i0.defineInjectable({ factory: function PartnersService_Factory() { return new PartnersService(); }, token: PartnersService, providedIn: "root" });
        return PartnersService;
    }());

    /**
     * @fileoverview added by tsickle
     * @suppress {checkTypes,extraRequire,uselessCode} checked by tsc
     */
    var PartnersComponent = (function () {
        function PartnersComponent() {
        }
        /**
         * @return {?}
         */
        PartnersComponent.prototype.ngOnInit = /**
         * @return {?}
         */
            function () {
            };
        PartnersComponent.decorators = [
            { type: i0.Component, args: [{
                        selector: 'lib-partners',
                        template: "\n    <p>\n      partners works!\n    </p>\n  ",
                        styles: []
                    },] },
        ];
        /** @nocollapse */
        PartnersComponent.ctorParameters = function () { return []; };
        return PartnersComponent;
    }());

    /**
     * @fileoverview added by tsickle
     * @suppress {checkTypes,extraRequire,uselessCode} checked by tsc
     */
    var NavigationComponent = (function () {
        function NavigationComponent() {
        }
        /**
         * @return {?}
         */
        NavigationComponent.prototype.ngOnInit = /**
         * @return {?}
         */
            function () {
            };
        NavigationComponent.decorators = [
            { type: i0.Component, args: [{
                        selector: 'lib-navigation',
                        template: "<p>\n  navigation works!\n</p>\n",
                        styles: [""]
                    },] },
        ];
        /** @nocollapse */
        NavigationComponent.ctorParameters = function () { return []; };
        return NavigationComponent;
    }());

    /**
     * @fileoverview added by tsickle
     * @suppress {checkTypes,extraRequire,uselessCode} checked by tsc
     */
    var PartnersModule = (function () {
        function PartnersModule() {
        }
        PartnersModule.decorators = [
            { type: i0.NgModule, args: [{
                        imports: [],
                        declarations: [
                            PartnersComponent,
                            NavigationComponent
                        ],
                        exports: [
                            PartnersComponent,
                            NavigationComponent
                        ]
                    },] },
        ];
        return PartnersModule;
    }());

    /**
     * @fileoverview added by tsickle
     * @suppress {checkTypes,extraRequire,uselessCode} checked by tsc
     */

    /**
     * @fileoverview added by tsickle
     * @suppress {checkTypes,extraRequire,uselessCode} checked by tsc
     */

    exports.PartnersService = PartnersService;
    exports.PartnersComponent = PartnersComponent;
    exports.NavigationComponent = NavigationComponent;
    exports.PartnersModule = PartnersModule;

    Object.defineProperty(exports, '__esModule', { value: true });

})));
