// partners/fesm2015/partners.js
import { Injectable, Component, NgModule, defineInjectable } from '@angular/core';

/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,uselessCode} checked by tsc
 */
class PartnersService {
    constructor() { }
}
PartnersService.decorators = [
    { type: Injectable, args: [{
                providedIn: 'root'
            },] },
];
/** @nocollapse */
PartnersService.ctorParameters = () => [];
/** @nocollapse */ PartnersService.ngInjectableDef = defineInjectable({ factory: function PartnersService_Factory() { return new PartnersService(); }, token: PartnersService, providedIn: "root" });

/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,uselessCode} checked by tsc
 */
class PartnersComponent {
    constructor() { }
    /**
     * @return {?}
     */
    ngOnInit() {
    }
}
PartnersComponent.decorators = [
    { type: Component, args: [{
                selector: 'lib-partners',
                template: `
    <p>
      partners works!
    </p>
  `,
                styles: []
            },] },
];
/** @nocollapse */
PartnersComponent.ctorParameters = () => [];

/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,uselessCode} checked by tsc
 */
class NavigationComponent {
    constructor() { }
    /**
     * @return {?}
     */
    ngOnInit() {
    }
}
NavigationComponent.decorators = [
    { type: Component, args: [{
                selector: 'lib-navigation',
                template: `<p>
  navigation works!
</p>
`,
                styles: [``]
            },] },
];
/** @nocollapse */
NavigationComponent.ctorParameters = () => [];

/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,uselessCode} checked by tsc
 */
class PartnersModule {
}
PartnersModule.decorators = [
    { type: NgModule, args: [{
                imports: [],
                declarations: [
                    PartnersComponent,
                    NavigationComponent
                ],
                exports: [
                    PartnersComponent,
                    NavigationComponent
                ]
            },] },
];

/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,uselessCode} checked by tsc
 */

/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,uselessCode} checked by tsc
 */

export { PartnersService, PartnersComponent, NavigationComponent, PartnersModule };
